Singularity

```diff
@@ -122,32 +102,23 @@ From: centos:7
     tar -xzf smalt-0.7.6-bin.tar.gz --no-same-owner
     ln -s /opt/smalt-0.7.6-bin/smalt_x86_64 /bin/smalt
 
+    echo ===== Installing Haploflow ===== >/dev/null
+    apt-get install -q -y libboost-all-dev build-essential sudo git ronn cmake
+    cd /opt/
+    git clone https://github.com/hzi-bifo/Haploflow
+    cd Haploflow
+    git checkout 9a5a0ff6c3a0435e723e41f98fe82ec2ad19cf50
+    yes | sh build.sh
```

Comment:
It looks like Haploflow installs itself into system folders, so you could probably reduce the image size by deleting the git clone.

Reply:
Ok, I will try!

Reply:
Actually, I just realized that this Haploflow installation wasn't in the original Singularity file. I will move this part to the Haploflow PR.